app/models/ingredient.rb

```diff
@@ -0,0 +1,2 @@
+class Ingredient < ApplicationRecord
+end
```

Comment:
Yes, thanks. I will add the following
validates :name, presence: true, uniqueness: true, length: { minimum: 2 }